examples/tcp-close/server.rs

```diff
@@ -175,6 +175,44 @@ impl TcpServer {
         Ok(())
     }
 
+    /// Runs the target TCP server which closes the sockets on connection.
+    pub fn run_close_sockets_on_accept(&mut self, nclients: Option<usize>) -> Result<()> {
+        // Accept new connection.
+        self.libos.listen(self.sockqd, nclients.unwrap_or(512))?;
+        self.issue_accept()?;
+
+        loop {
+            // Stop when enough connections have been terminated.
+            if let Some(nclients) = nclients {
+                if self.clients_closed >= nclients {
+                    // Sanity check that all connections have been closed.
+                    const ERR_MSG: &str = "there should be no clients connected, but there are";
+                    assert_eq!(self.clients.len(), 0, "{}", ERR_MSG);
+                    break;
+                }
+            }
+
+            let (_index, qr): (usize, demi_qresult_t) = self.libos.wait_any(&self.qts, None)?;
```

Comment:
We are missing to `remove()` the completed queue token here.

Reply:
Done